PSK.Model/Services/InviteService.cs

```diff
@@ -0,0 +1,76 @@
+﻿using PSK.Model.DBConnection;
+using PSK.Model.Entities;
+using System;
+using System.Net.Mail;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace PSK.Model.Services
+{
+    public class InviteService : IInviteService
+    {
+        MockDBConnection db = new MockDBConnection();
```

Comment:
Šitą reikia įsi'injectint per serviso konstruktorių (kaip ir į controlerį įsi'injectinai patį InviteService).